### DISCUSSION TITLE
LightWeightChart minute or hourly timeframe

### DISCUSSION THREAD

**Discussion by benhoben** [tradingview/lightweight-charts#1256] [Q&A]
I have a daily chart on lightweight chart.

However, cannot find a way to change it to minute time frame in the same date, please help, thank you :)

**Comment by SlicedSilver** [comment]
You can set `timeVisible` to `true` in the `TimeScaleOptions`. Documentation: https://tradingview.github.io/lightweight-charts/docs/api/interfaces/TimeScaleOptions#timevisible

```js
const chart = createChart(chartElement, {
  timeScale: {
    timeVisible: true,
  },
});
```

**Comment by romancerny** [comment]
Thanks, this solution shows time, but all hourly candles draw in one place of the same day. Would you have an example of hourly chart? All available examples are only for daily chart.

On your website you state: 
"Time intervals & date ranges
Our charts can easily display custom intervals - such as 5 hours, 7 minutes or 15 second bars.
Custom S/M/H/D/W/Mo/Y"

but there is no way I can achieve that.

Please advise

Is that a bug?

**Comment by romancerny** [comment]
Found a solution on Discord. It only works with Unix epoch timestamps.

**Comment by SlicedSilver** [comment]
Here is an example: https://codesandbox.io/p/sandbox/lightweight-charts-js-starter-forked-pwghsb?file=%2FREADME.md

**Comment by romancerny** [comment]
Thank you, that's very helpful
